appveyor.yml

```diff
@@ -17,6 +17,7 @@ install:
   - set PATH=%APPDATA%\npm;%APPVEYOR_BUILD_FOLDER%\node_modules\.bin;%PATH%
   - git submodule update --init --recursive
   - npm i
+  - SET
```

Comment:
@vweevers Can we leave this in or should we remove it?

Reply:
We don't need it anymore, let's remove it for cleaner logs